src/datatrove/pipeline/readers/parquet.py

```diff
@@ -33,6 +33,7 @@ def __init__(
         self,
```

Comment:
Could you also add the limit section in docstring above ?

Reply:
Yes, done, thank you for the quick review!